CHANGELOG.md

```diff
@@ -27,6 +27,7 @@
 
 - `filereceiver`: Add basic file reading capabilities to file receiver. Second PR. (#14638)
   This component is not yet enabled.
+- `s3exporter`: Add aws s3 exporter, config + exporter stub (#9979).
```

Comment:
`CHANGELOG.md` must not be modified directly, create a changelog note following [the CONTRIBUTING.md instructions](https://github.com/open-telemetry/opentelemetry-collector/blob/main/CONTRIBUTING.md#adding-a-changelog-entry)

Reply:
@mx-psi done.